requirements/tests.txt

```diff
@@ -1,8 +1,8 @@
-coverage==4.5.1
-isort==4.3.4
-pycodestyle==2.4.0
-pydocstyle==2.1.1
-pylint==1.9.1
-pytest==3.6.0
-pytest-cov==2.5.1
-pytest-pylint==0.9.0
+coverage==5.5
+isort==5.8.0
+pycodestyle==2.7.0
+pydocstyle==6.0.0
+pylint==2.7.4
+pytest==6.2.3
+pytest-cov==2.11.1
+pytest-pylint==0.18.0
```

Comment:
How did you proceed to upgrade pip package versions? Remove version pins, run `pip3 install --upgrade -r requirements/tests.txt` ?